src/machine.rs

```diff
@@ -227,16 +225,19 @@ impl OlmMachine {
     /// "changed" notification for that user in the future.
     ///
     /// Users that were already in the list are unaffected.
+    ///
+    /// `users` is going to be dropped on the JS part after the function
+    /// returns. Be careful to not used `users` after that.
```

Comment:
"dropped" isn't really a term that means anything to the average JS developer. Could you perhaps phrase this in a way that will be easier to understand?

TBH I'm slightly confused myself. Is it the individual `UserID` objects, or the array, that will be invalidated? Or both?

Reply:
The individual `UserId` objects. That's how `wasm-bindgen` works. It doesn't make me super happy, but it is what it is.

I'll update the doc'.

Reply:
Also: I feel like we should have a similar warning on all the other methods which consume a `Vec`, like `request_verification`.